app/src/main/sentences/en/navigate.dslf

```diff
@@ -0,0 +1,2 @@
+navigate: high
+(navigate to?)|(take me to)|(directions to?)|(where s|is) .where.;
```

Comment:
A few more ways to say things
```suggestion
(give? me? directions|direction|navigate to?)|(take|bring me to)|(where s|is) .where.;
```

Reply:
Wouldn't this also accept a command of "give navigate <address>"? Or "give me navigate to <address>"? Seems undesirable.

Reply:
I went ahead and added a modified version of your suggestion. Will resolve this conversation, but feel free to re-open.